app/components/icon_list_component.rb

```diff
@@ -0,0 +1,35 @@
+class IconListComponent < BaseComponent
+  renders_many :items, ->(**kwargs, &block) {
+    IconListItemComponent.new(icon:, color:, **kwargs, &block)
+  }
```

Comment:
I feel like we mostly use `do....end` for multi-line blocks and I confirmed with `ruby-parse` that this gives us the same results (sometimes ommitting parens means the block gets passed to `renders_many` instead of to the proc)

```suggestion
  renders_many :items, ->(**kwargs, &block) do
    IconListItemComponent.new(icon:, color:, **kwargs, &block)
  end
```

Reply:
Yeah agreed 👍

Reply:
Looks like we've got the [`do ... end` syntax in `StatusPageComponent`](https://github.com/18F/identity-idp/blob/32acb071e03cc6aa4ce68fb5deeb22a0b47d8078/app/components/status_page_component.rb#L11-L13), so I should expect that will work. I admit that I largely copied from `ProcessListComponent` for this, which [uses the curly braces](https://github.com/18F/identity-idp/blob/32acb071e03cc6aa4ce68fb5deeb22a0b47d8078/app/components/process_list_component.rb#L2-L4).

In 1f71d2e8f, I've updated them all to use the same multiline syntax.